Tribler/Test/Core/Modules/RestApi/test_my_channel_endpoints.py

```diff
@@ -110,15 +217,15 @@ def test_rss_feeds_endpoint_no_my_channel(self):
 
 
 class TestMyChannelRssEndpoints(AbstractTestMyChannelEndpoints):
-
```

Comment:
Very minor detail: I think it's a bit clearer to leave the newlines after a class definition, but please feel free to leave it if you want :)